### DISCUSSION TITLE
How do you remove certificates?

### DISCUSSION THREAD

**Discussion by Zerowalker** [go-acme/lego#1817] [Q&A] [answered]
I got some older certificates which I would like to remove from lego.
I also go on older ".lego" folder with an account which i want removed as well, not sure if i can just delete the folder or not.
I kinda messed up between the local accounts so ended up with some duplicate lego setups and want to clean up so only the actual one and my current certificate are left.

**Comment by ldez** [comment, marked as answer]
Hello,

you can remove certificates inside the directory `.lego/certificates/`.
you can also remove old accounts inside the directory `.lego/accounts/`.

**Reply by Zerowalker** [reply]
so it's just to remove them?
i got an "expiration" mail from one of these older certificates, assuming that  just happens one time per renew though.

**Reply by ldez** [reply]
You can also revoke certificates but you have to use the same account used to create the certificates.

https://go-acme.github.io/lego/usage/cli/options/

**Reply by Zerowalker** [reply]
yeah i can do that with one of them, but it doesn't remove them.
But i will just delete the files/folders as you suggested:)
